themes/edx.org/lms/templates/header.html

```diff
@@ -104,6 +104,10 @@ <h2 class="course-header">
 
       <%include file="user_dropdown.html"/>
 
+      <a href="${get_online_help_info(online_help_token)['doc_url']}" 
```

Comment:
As we discussed in hipchat, @AlasdairSwan suggested that I not wrap the link in a list element. Alasdair, please let me know if I'm misremembering.